### PR TITLE
Closes #77 - Add support for scores, skips, and same position matchin…

### LINKED ISSUE #77: Add support for scores, skips and same position matching for content queries (adjacent, within, phrase)

### PR DESCRIPTION
**Pull request by drewfarris**
…g for content queries(adjacent, within, phrase)

### CODE REVIEW

File: warehouse/query-core/src/test/java/datawave/query/jexl/functions/ContentFunctionQueryTest.java [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+        // create and process events with test data
+        RawRecordContainer record = new RawRecordContainerImpl();
+        Multimap<BulkIngestKey,Value> keyValues = HashMultimap.create();
+        
+        record.clear();
+        record.setDataType(new Type(DATA_TYPE_NAME, TestIngestHelper.class, null, null, 1, null));
+        record.setRawFileName("example.dat");
+        record.setRawRecordNumber(1);
+        record.setDate(formatter.parse(BEGIN_DATE).getTime());
+        record.setRawDataAndGenerateId(data);
```
Comment (pcgrenier) [inline warehouse/query-core/src/test/java/datawave/query/jexl/functions/ContentFunctionQueryTest.java]: This method seems to have been refactored into two calls.
record.setRawData(data);
record.generateId(null);

**Comment by drewfarris** [comment]
test webhook

**Comment by drewfarris** [comment]
@burrito-czar ok to test

**Comment by drewfarris** [comment]
@burrito-czar ok to test

**Comment by drewfarris** [comment]
@burrito-czar test this please

**Pull request merged**